src/main/resources/templates/addRecipePage.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="pl" xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout}">
+<main layout:fragment="content">
+<img class="rounded mx-auto d-block" width="400" alt="logo" th:src="@{/images/addRecipe.png}"> </br>
+<p><strong th:if="${newRecipe.id} ne null" th:text="|Edycja przepisu: ${newRecipe.name}|"></strong></p>
+<form method="post" th:action="@{|/add-recipe?recipeid=${newRecipe.id}|}" th:object="${newRecipe}">
+    <div class="mb-3">
+        <label for="recipeName" class="form-label">Nazwa przepisu</label>
+        <input type="text" th:field="*{name}" id="recipeName" placeholder="Nazwa przepisu" class="form-control">
+    </div>
+    <div class="mb-3">
+        <label for="recipeDesc" class="form-label">Tresc przepisu</label>
+        <textarea th:field="*{description}" class="form-control" id="recipeDesc" rows="5" placeholder="Przepis"></textarea>
+    </div>
+    <label for="categorySelect" class="form-label">Kategoria:</label>
+    <select th:field="*{category}" class="form-select" aria-label="Disabled select example" id="categorySelect">
+        <option th:each="categoryItem : ${categoriesList}" th:text="${categoryItem.name}" th:value="${categoryItem.id}"></option>
+    </select>
+  <button type="submit">Zapisz</button>
```

Comment:
Z dodaną klasą `btn btn-primary mt-1` będzie ładniej (mt to margin-top, wartość można zwiększyć do 2, 3 itd.)